File: pickem-server/PickEmServer/Api/Models/League.cs
using PickEmServer.App;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace PickEmServer.Api.Models
{
    public class League
    {
        public int CurrentWeekRef { get; internal set; }
        public string LeagueCode { get; internal set; }
        public string LeagueTitle { get; internal set; }
        public string NcaaSeasonCodeRef { get; set; }
        public PickemScoringTypes PickemScoringType { get; set; }
        public string SeasonCodeRef { get; set; }
    }
}

File: pickem-web/src/app/sub-system/models/api/league.ts
import { PickemScoringTypes } from "./enums";

export class League
{
    currentWeekRef: number;
    leagueCode: string;
    leagueTitle: string;
    ncaaSeasonCodeRef: string;
    pickemScoringType: PickemScoringTypes;
    seasonCodeRef: string;
}